gax-httpjson/src/main/java/com/google/api/gax/httpjson/HttpRequestRunnable.java

```diff
@@ -50,13 +50,15 @@
 import java.util.LinkedList;
 import java.util.List;
 import java.util.Map;
+import java.util.Map.Entry;
+import org.apache.http.client.utils.URIBuilder;
```

Comment:
We shouldn't add a new dependency just to use one class. New dependencies invite new possibilities for diamond dependency conflicts.

Reply:
Ok, removing that dependency.
Reverting back to using GenericUrl.set(String, Object) for setting query params. The [unit tests](https://github.com/google/google-http-java-client/blob/98e6d4865c85da713b64b05a99baaad193ba07c6/google-http-client/src/test/java/com/google/api/client/http/GenericUrlTest.java) for GenericUrl imply that if the Object param is a List<String>, the resulting url.toString() will work as expected and have one query param for each of the values in the list.